src/code42cli/cmds/devices.py

```diff
@@ -206,51 +210,78 @@ def _get_device_info(sdk, device_guid):
     is_flag=True,
     help="""Include device settings in output.""",
 )
+@click.option(
+    "--last-connected-before",
+    type=MagicDate(rounding_func=round_datetime_to_day_start),
+    help=f"Include devices only when the 'lastConnected' field is after provided value. {MagicDate.HELP_TEXT}",
```

Comment:
For these helps, should be "is after _the_ provided value" , missing the's

Reply:
updated